src/components/Views.tsx

```diff
@@ -78,8 +85,18 @@ const OverwriteView = ({ templateName, onCancel, onOverwrite }: TOverwriteViewPr
     </div>
 );
 
-const ActiveListView = ({ views, removeAll, applyLayout, remove }: TActiveListViewProps) => {
+const ActiveListView = ({ allowTickChartTypeOnly, views, removeAll, applyLayout, remove }: TActiveListViewProps) => {
     if (!views.length) return null;
+    const isDisabled = (layout: TMainStore['view']['sortedItems'][number]['layout']) => {
+        const { chartType, periodicity, timeUnit } = layout ?? {};
+        const oneTickChartTemplate = chartType === 'mountain' && periodicity === 1 && timeUnit === 'second';
+        return allowTickChartTypeOnly && !oneTickChartTemplate;
+    };
+    const sorted_views = [...views].sort((a, b) => {
```

Comment:
if you don't want to mutate `views` array, you can use `.toSorted()` method (`.toSorted()` method of Array instances is the copying version of the `.sort()` method; it returns a new array with the elements sorted) instead of creating copy and sorting it

Reply:
A got it, it's not supported in SmartCharts